_component_code/footer.md

```diff
@@ -9,3 +9,57 @@ collection: component_code
 ---
 
 {% include todo.html content="Set up code." %}
+
+# Footer
+
+The footer lives at the bottom of the page to help visitors find
+globally applicable, but non-primary navigation information. We
+recommend using it on every page and every wrapped application. For
+customer applications, it  should not exceed the articulated scope.
+
+## When to use footers
+
+Use the Rackspace footer on every application page to provide consistent
+global functionality on all of our platforms.
+
+## Best practices for footers
+
+Ensure the footer is placed at the bottom of the page, and flush to the right
+of any left-hand navigation, spanning the length of the screen. Ensure the
+footer contains the following:
+
+* Rackspace copyright and year
+* A link to website terms and conditions
+* A link to the privacy policy
+
+# Specifications
+
+## Footer scope
+
+Place the footer at the bottom of the navigational page.
+It should also be flush to the left navigation or, if using top navigation,
+the footer container goes across the bottom of the entire page. The footer
+must contain copyright information with links to Rackspace website terms
+and privacy policy.
+
+* Website Terms: https://www.rackspace.com/information/legal/websiteterms
+* Privacy Policy: https://www.rackspace.com/information/legal/privacystatement
+
+For customer applications, do not add additional functionality to the footer.
+
+{% figure [caption:"The footer rests on the bottom of whatever page it
+  is on."] [class:"image bg-light border"] %}
+  ![]({{site.url}}/assets/images/components/content-areas/footer/footer-scope.svg){:width="80%"}
+{% endfigure %}
+
+## Footer specifications
+
+All footer content starts after the gutter at 20px. The footer container is
+40px tall with vertically centered text. Items are placed 32px between each
+other. Links behave according to the Links documentation, utilizing cyan 500
```

Comment:
Links are colored cyan 500 on dark backgrounds and behave as described in the Links documentation.